domains/saqlain.json

```
@@ -4,6 +4,6 @@
     "email": "saqlainmohd639@gmail.com"
   },
   "records": {
-    "CNAME": "mohdsaqlain.vercel.app"
+    "A": "216.198.79.1"
```

A records is an instance of Array
```suggestion
    "A": ["216.198.79.1"]
```